### ISSUE TITLE
Saving error on Protégé since Annotation Properties section

### ISSUE THREAD

**Issue by legallm** [IHTSDO/snomed-owl-toolkit#88]
Hello,
Since the 1st March International release, the OWL file generated by the converter contains an *Annotation Properties* section:
```
############################
#   Annotation Properties
############################

# Annotation Property: <http://snomed.info/id/1295447006> (Annotation attribute (attribute))

AnnotationAssertion(rdfs:label :1295447006 "Annotation attribute (attribute)"@en)
AnnotationAssertion(skos:prefLabel :1295447006 "Annotation attribute"@en)

# Annotation Property: <http://snomed.info/id/1295448001> (Attribution (attribute))

AnnotationAssertion(rdfs:label :1295448001 "Attribution (attribute)"@en)
AnnotationAssertion(skos:definition :1295448001 "Attributed to a particular organisation or group that contributes content to SNOMED CT."@en-gb)
AnnotationAssertion(skos:definition :1295448001 "Attributed to a particular organization or group that contributes content to SNOMED CT."@en-us)
AnnotationAssertion(skos:prefLabel :1295448001 "Attribution"@en)
SubAnnotationPropertyOf(:1295448001 :1295447006)

# Annotation Property: <http://snomed.info/id/1295449009> (Additional relationship attribute (attribute))

AnnotationAssertion(rdfs:label :1295449009 "Additional relationship attribute (attribute)"@en)
AnnotationAssertion(skos:definition :1295449009 "A grouper concept for annotation attributes that are represented as additional relationships."@en)
AnnotationAssertion(skos:prefLabel :1295449009 "Additional relationship attribute"@en)
SubAnnotationPropertyOf(:1295449009 :1295447006)
```
However, it seems that this section causes issues while trying to save the file in RDF/XML through Protégé:
<img width="236" alt="Protege_save_error" src="https://github.com/IHTSDO/snomed-owl-toolkit/assets/75123969/aa18b173-7729-42ba-8ac6-e0c9e6daffa2">
```
2024-03-04 10:57:02.594 [AWT-EventQueue-0] ERROR OWLEditorKit    An error occurred whilst saving the http://snomed.info/id/1295447006
http://snomed.info/id/1295449009
http://snomed.info/id/1295448001 ontology: {}
org.semanticweb.owlapi.model.OWLOntologyStorageException: http://snomed.info/id/1295447006
http://snomed.info/id/1295449009
http://snomed.info/id/1295448001
	at org.semanticweb.owlapi.rdf.rdfxml.renderer.RDFXMLStorer.storeOntology(RDFXMLStorer.java:57) ~[na:na]
	at org.semanticweb.owlapi.util.AbstractOWLStorer.store(AbstractOWLStorer.java:99) ~[na:na]
	at org.semanticweb.owlapi.util.AbstractOWLStorer.storeOntology(AbstractOWLStorer.java:64) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.saveOntology(OWLOntologyManagerImpl.java:1486) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.OWLImmutableOntologyImpl.saveOntology(OWLImmutableOntologyImpl.java:1459) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.concurrent.ConcurrentOWLOntologyImpl.lambda$saveOntology$12(ConcurrentOWLOntologyImpl.java:369) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.concurrent.ConcurrentOWLOntologyImpl.callReadLock(ConcurrentOWLOntologyImpl.java:196) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.concurrent.ConcurrentOWLOntologyImpl.saveOntology(ConcurrentOWLOntologyImpl.java:369) ~[na:na]
	at org.protege.editor.owl.model.io.OntologySaver.saveOntologyInternal(OntologySaver.java:96) ~[na:na]
	at org.protege.editor.owl.model.io.OntologySaver.lambda$saveOntologies$0(OntologySaver.java:52) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_121]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_121]
Caused by: org.semanticweb.owlapi.rdf.rdfxml.renderer.IllegalElementNameException: Illegal Element Name (Element Is Not A QName): http://snomed.info/id/1295447006
http://snomed.info/id/1295449009
http://snomed.info/id/1295448001
	... 14 common frames omitted
```

**Comment by kaicode** [comment]
I've reviewed the OWL Toolkit code changes in this area. I've generated the International February and March OWL files, inspected the files and loaded them in to Protege. I've reproduced the RDF/XML conversion error with the March release.

The error seems to suggest that the annotations within the ontology are using a full URL as an identifier (IRI) rather than something simpler that would be acceptable as an XML element name, for example a CamelCaseName. It's odd that this is happening only with annotations when we already have object and data properties that seem to be working fine. They are all using the same conventions for identification.

As a test I tried manually changing the IRI of the three annotations using Protege:
<img width="688" alt="Screenshot 2024-03-17 at 00 21 01" src="https://github.com/IHTSDO/snomed-owl-toolkit/assets/2010517/b0b2d6c5-5dfe-49cc-aafb-d15cd89dc806">

This seems to resolve the issue; the ontology can then be saved as RDF/XML.
This replacement could be automated using a script.
